.github/workflows/profiling-benchmarks.yml

```
@@ -0,0 +1,217 @@
+
+    - name: Comment on commit (if triggered manually)
+      if: github.event_name == 'workflow_dispatch'
+      run: |
+        echo "Manual profiling run completed for commit ${{ github.sha }}"
+        echo "Check the Actions tab for detailed results and artifacts"
+
+  memory-stress-test:
+    name: Memory Stress Testing
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    needs: []  # Run independently
```

actionlint (1.7.7)

"needs" section should not be empty

(syntax-check)